test/e2e/pages/index.js

```diff
@@ -16,4 +17,5 @@ export {
   movesDashboardPage,
   createMovePage,
   cancelMovePage,
+  UpdateMovePage,
```

Comment:
I wonder if we should use the same pattern that already exists and instantiate the class above and export an instantiated object here.

Reply:
But it needs to be different for each move id. If the tests always run sequentially, I suppose you could just set the `this.url` value

Reply:
It's probably best not to assume they'll run sequentially in that case